lib/modules/manager/bazel/rules/docker.ts

```diff
@@ -0,0 +1,38 @@
+import is from '@sindresorhus/is';
+import { DockerDatasource } from '../../../datasource/docker';
+import { id as dockerVersioning } from '../../../versioning/docker';
+import type { PackageDependency } from '../../types';
+import type { Target } from '../types';
+
+export function dockerDependency({
+  rule: depType,
+  name: depName,
+  tag: currentValue,
+  digest: currentDigest,
+  repository: packageName,
+  registry,
+}: Target): PackageDependency | null {
+  let dep: PackageDependency | null = null;
+
+  if (
```

Comment:
can we invert the condition, makes code flow easier

Reply:
Let's do in separate PR